partner-compatibility-testing.md

```diff
@@ -0,0 +1,20 @@
+# Introduction
+We have marked several tests with label `PartnerComp`.
+Those tests can be used in order to test the compatibility of a backup-restore solution with the kubevirt project.
+
+# Prerequisites
+Have a cluster ready to run backup and restore. Make sure [KubeVirt](https://github.com/kubevirt/kubevirt) and [CDI](https://github.com/kubevirt/containerized-data-importer) are installed.
+
+# Requirements
+Make a copy of the [template script](cmd/backup-restore-script-template/main.go).
```

Comment:
Typo: Shouldn't be `template-script-backup-restore/main.go` instead of `backup-restore-script-template/main.go`?

Reply:
you are obviously right :)